From 6c5364736f5afd1106ac240f03806af53979cc60 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Mon, 25 Feb 2019 15:56:50 +0800
Subject: [PATCH 22/24] include sys/wait.h to avoid compile failure

Fix the following error:
src/udev/udev-event.c:581:53: error: 'WEXITED' undeclared (first use in this function); did you mean 'WIFEXITED'?

Upstream-Status: Inappropriate [musl specific]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/udev/udev-event.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/udev/udev-event.c b/src/udev/udev-event.c
index 07b7365..e532bb7 100644
--- a/src/udev/udev-event.c
+++ b/src/udev/udev-event.c
@@ -8,7 +8,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
-
+#ifndef __GLIBC__
+#include <sys/wait.h>
+#endif
 #include "sd-event.h"
 
 #include "alloc-util.h"
-- 
2.7.4

